### PR TITLE
Status endpoint

### PR DESCRIPTION
**Pull request by lukitsbrian**
https://github.com/allinbits/demeris-backend/issues/403

### CODE REVIEW

File: api/chains/chains.go [hunk 1]
```diff
@@ -665,45 +665,18 @@ func GetChainStatus(c *gin.Context) {
...
-			"cannot retrieve chain last block time",
-			"id",
-			e.ID,
-			"name",
-			chainName,
-			"error",
-			err,
-		)
-
+		c.JSON(http.StatusBadRequest, res)
```
Comment (DeshErBojhaa) [inline api/chains/chains.go:672]: Don't we want to set `chain.Online = false` in this case?

File: api/chains/chains_test.go [hunk 2]
```diff
@@ -155,3 +166,75 @@ func toSupportedChain(c cns.Chain) chains.SupportedChain {
...
+			true,
+		},
+		// {
+		// 	"Get Chain Status - Enabled",
+		// 	chainWithPublicEndpoints,
+		// 	chainWithPublicEndpoints.ChainName,
+		// 	200,
+		// 	chains.StatusResponse{Online: true},
+		// 	true,
+		// },
```
Comment (DeshErBojhaa) [inline api/chains/chains_test.go]: Why ignore?

File: api/chains/chains_test.go [hunk 3]
```diff
@@ -155,3 +166,75 @@ func toSupportedChain(c cns.Chain) chains.SupportedChain {
...
+			disabledChain.ChainName,
+			400,
+			chains.StatusResponse{Online: false},
+			true,
+		},
+	}
+
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+
```
Comment (DeshErBojhaa) [inline api/chains/chains_test.go]: `t.Parallel()` for faster exec and real world simu. Also consider for the function above.

**Review by DeshErBojhaa — APPROVED** [review]
mostly LGTM, after you address my comments.

File: api/chains/chains.go (hunk 1 above)
Comment (pitasi) [reply api/chains/chains.go:672]: I believe that "false" is the default value for bools

Comment (pitasi) [reply api/chains/chains.go:672]: but I saw he explictly set it to false below, so maybe we can be consistent and add it here too

**Review by DeshErBojhaa — APPROVED** [review]

**Review by pitasi — APPROVED** [review]

**Pull request merged**